.tekton/marin3r-operator-bundle-push.yaml

```diff
@@ -26,6 +36,16 @@ spec:
     value: quay.io/redhat-user-workloads/api-management-tenant/marin3r-operator-bundle:{{revision}}
   - name: dockerfile
     value: Containerfile.marin3r-operator-bundle
+  - name: hermetic
+    value: "true"
+  - name: prefetch-input
+    value:
+     # - {"type": "rpm", "path": "."}
```

Comment:
same as above

Reply:
it was just to check ... 
The problem was:
2025-05-06 11:26:12,485 ERROR UnsupportedFeature: Package manager(s) not yet supported: rpm
Error: UnsupportedFeature: Package manager(s) not yet supported: rpm